libraries/provision/ansible/playbooks/tasks/stop-sg-accel.yml

```diff
@@ -1,11 +1,11 @@
 # Stop service
 - name: SG ACCEL | stop sg_accel service
   service: name=sg_accel state=stopped
-  when: ansible_distribution == "CentOS" and ansible_distribution_major_version == "7"
+  when: (ansible_distribution == "CentOS" and ansible_distribution_major_version == "7") or (ansible_distribution == "Ubuntu") or ansible_distribution == "RedHat" or ansible_distribution == "Amazon"
 
-- name: SG ACCEL | stop sg_accel for CentOS 6
+- name: SG ACCEL | stop sg_accel
```

Comment:
This stop sg_accel did not work on debian. I believer it is same for ubuntu. Can you double check with commenting any code in tear down?